js/src/components/google-ads-account-card/google-ads-account-card.test.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import GoogleAdsAccountCard from './google-ads-account-card';
+import useGoogleAdsAccountStatus from '.~/hooks/useGoogleAdsAccountStatus';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import useGoogleAccount from '.~/hooks/useGoogleAccount';
+
+jest.mock( '.~/hooks/useGoogleAdsAccountStatus', () => ( {
+	__esModule: true,
+	default: jest.fn().mockName( 'useGoogleAdsAccountStatus' ),
+} ) );
+
+jest.mock( '@woocommerce/components', () => ( {
+	...jest.requireActual( '@woocommerce/components' ),
+	Spinner: jest
+		.fn( () => <div role="status" aria-label="spinner" /> )
+		.mockName( 'Spinner' ),
+} ) );
+
+jest.mock( '.~/hooks/useGoogleAdsAccount', () =>
+	jest.fn().mockName( 'useGoogleAdsAccount' ).mockReturnValue( {} )
+);
+
+jest.mock( '.~/hooks/useGoogleAccount', () =>
+	jest.fn().mockName( 'useGoogleAccount' ).mockReturnValue( {} )
+);
+
+jest.mock( '.~/hooks/useDispatchCoreNotices', () => ( {
+	__esModule: true,
+	default: jest
+		.fn()
+		.mockName( 'useDispatchCoreNotices' )
+		.mockImplementation( () => {
+			return {
+				createNotice: jest.fn(),
+			};
+		} ),
+} ) );
+
+const CONNECTED_GOOGLE_ADS_ACCOUNTS = {
```

Comment:
Probably it would be better to use the singular noun.

Reply:
Sure! Updated!